charts/kube-ovn/templates/central-deploy.yaml

```diff
@@ -74,6 +74,8 @@ spec:
               value: "{{- .Values.func.ENABLE_BIND_LOCAL_IP }}"
             - name: PROBE_INTERVAL
               value: "{{ .Values.networking.PROBE_INTERVAL }}"
+            - name: OVN_NORTHD_PROBE_INTERVAL
+              value: "{{ .Values.networking.OVN_NORTHD_PROBE_INTERVAL}}"
```

Comment:
We need to add a new value in `values.yaml`.

Reply:
![image](https://github.com/kubeovn/kube-ovn/assets/47097611/60e26c03-6167-4c8a-b833-9b9c5baa7633)